source/index.txt

```diff
@@ -22,16 +16,60 @@ The Node driver documentation is organized as follows:
   /api-documentation
   /faq
   /issues-and-help
+  /compatibility
 
-Compatibility
--------------
 
-MongoDB Compatibility
-~~~~~~~~~~~~~~~~~~~~~
+Introduction
+------------
+The official MongoDB Node.js driver allows Node.js applications to connect and perform queries with MongoDB.
+
+.. note::
+   
+   These docs are for version 3.5 of the MongoDB Node.js driver. If
+   you're looking for an older version of the MongoDB Node.js driver
+   docs, `see the GitHub pages
+   <https://mongodb.github.io/node-mongodb-native/>`_. For the main
+   MongoDB documentation, see the :manual:`MongoDB Manual </>`.
+
+Quick Start
+-----------
+Learn how to quickly get a Node.js application working with MongoDB Atlas through the `Quick Start </quick-start/>`_ section
+
+Fundamentals
+------------
+The `Fundamentals </fundamentals>`_ section defines the following core concepts involved with using the MongoDB
+Node.js driver: 
 
-.. include:: /includes/mongodb-compatibility-table-node.rst
+* CRUD: Learn how to read and write data to MongoDB using Node.js
+* Authentication: Learn how to log a user in through various community and enterprise mechanisms
```

Comment:
"various community and enterprise mechanisms" doesn't mean much if I'm a brand new user checking out the fundamentals. Maybe: "learn how to configure authentication and log a user in"

Reply:
fair enough, changed to:

"Learn how to configure authentication and log a user in using the MongoDB Node.js driver"